tsl/src/compression/compression_scankey.c

```diff
@@ -335,6 +337,7 @@ build_index_scankeys_using_slot(Oid hypertable_relid, Relation in_rel, Relation
 
 			if (isnull)
 			{
+				*index_columns = bms_add_member(*index_columns, column_attno);
```

Comment:
Would be good to cover this line as well.

Reply:
Adding a null check test.